eng/pipelines/common/templates/runtimes/run-test-job.yml

```diff
@@ -62,6 +62,9 @@ jobs:
       - ${{ if ne(parameters.liveLibrariesBuildConfig, '') }}:
         - ${{ format('libraries_build_{0}{1}_{2}_{3}', parameters.osGroup, parameters.osSubgroup, parameters.archType, parameters.liveLibrariesBuildConfig) }}
 
+    - ${{ if eq(parameters.runtimeFlavor, 'mono') }}:
+      - ${{ format('mono_product_build_{0}{1}_{2}_{3}', parameters.osGroup, parameters.osSubgroup, parameters.archType, coalesce(parameters.liveRuntimeBuildConfig, parameters.buildConfig)) }}
```

Comment:
We don't need to use the coalesce here, liveRuntimeBuildConfig is referring to `coreclr's` build config, not mono's, right?

Reply:
Removed this any way.